### PR TITLE
chore(ui-kit): add top-level storybook link to docs

### PR DESCRIPTION
**Pull request by jaikamat**
## Summary

This PR adds a top-level Storybook link to the UI Kit documentation

## Description

A nonzero number of developers (myself included 😅 ) have reached out to ask where the link to the UI Kit Storybook is, and I've often found myself landing on this page finding it difficult to find it 🔍 .

I figured we could add one at the top level, but as this package is open source, I was unsure of any formalities surrounding editing the documentation here. I'm mostly just throwing this PR at the wall to see if it sticks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 > If you are building **Custom Applications** for the Merchant Center, be sure to check out our [documentation](https://docs.commercetools.com/custom-applications)
 
+> An interactive collection of UI Kit components can be found [in our Storybook](https://uikit.commercetools.com/?path=/story/introduction--getting-started)
```
Comment (ByronDWall) [inline README.md]:
```suggestion
> Interactive documentation of UI Kit components can be found [in our Storybook](https://uikit.commercetools.com/?path=/story/introduction--getting-started)
```

**Review by ByronDWall — APPROVED** [review]
I think having this in the main readme would be really helpful and makes a lot of sense, I've left a suggestion as to phrasing, but other than that lgtm!

**Comment by emmenko** [comment]
Thanks for the PR.

One question first: isn't the link in the repo visible enough?

image

**Comment by jaikamat** [comment]
@emmenko Definitely visible, but if we feel strongly about not adding a link here's a suggestion! When a user clicks on the UI Kit top big image, it just links back to the Github repo. What if it linked to Storybook?

**Review by emmenko — APPROVED** [review]
All good, was just curious about the URL link in the github repo if it was visible or not.

I'm fine with the link in the readme, thanks.

**Review by tdeekens — APPROVED** [review]
🎉 First contribution 👯

**Comment by jaikamat** [comment]
It appears I'm not authorized to merge this PR 😬  What should I do?

**Pull request merged**